src/core/App/theme.js

```diff
@@ -7,13 +7,12 @@ export const theme = {
         pattensBlue: "#D6E4FF",
         mercury: "#E5E5E5",
         black: "#18181B",
```

Comment:
Why white is deleted?

Reply:
because white was declared twice so I removed one